### ISSUE TITLE
Node Engine Expected version ">12". Got "12.19.0"

### ISSUE THREAD

**Issue by frederikocmr** [apiaryio/s3-streaming-upload#72]
In my package .json I have the engine configured as 12.19.0. Whenever I try to install this package, this error appears.
Please ensure that the package works on all versions of node!

```
error s3-streaming-upload@0.3.2: The engine "node" is incompatible with this module. Expected version ">12". Got "12.19.0"
error Found incompatible module.
```

**Comment by abtris** [comment]
I will check it. Which npm version?

**Comment by frederikocmr** [comment]
I am currently using **yarn**.

But here is the versions that I am using:
`Npm: 6.14.8`
`Yarn: 1.22.5`

**Comment by abtris** [comment]
ok, I get just warning.

```
npm i s3-streaming-upload                                                                                                                                                                                                                                                                                           
npm WARN notsup Unsupported engine for s3-streaming-upload@0.3.2: wanted: {"node":">12"} (current:{"node":"12.13.0","npm":"6.14.8"})
```

**Comment by abtris** [comment]
Fixed in 0.3.3
